t/open311/getupdates.t

```diff
@@ -236,15 +244,22 @@ subtest 'test translation of auto-added comment from old-style Open311 update' =
     FixMyStreet::override_config {
         ALLOWED_COBRANDS => [ 'fixamingata' ],
     }, sub {
-        ok $updates->update_reports( [ 638346 ], $o, $body ), 'Updated reports';
+        my $updates = Open311::GetUpdates->new(
+            system_user => $user,
+            current_open311 => $o,
+            current_body => $body,
+            blank_updates_permitted => 1,
+        );
+        $updates->update_reports( [ $problem3 ] );
     };
     my @parts = uri_split($o->test_uri_used);
     is $parts[2], '/requests.xml', 'path matches';
     my @qs = sort split '&', $parts[3];
     is_deeply(\@qs, [ 'jurisdiction_id=mysociety', 'service_request_id=638346' ], 'query string matches');
 
     is $problem3->comments->count, 1, 'added a comment';
-    is $problem3->comments->first->text, "Stängd av kommunen", 'correct comment text';
+    is $problem3->comments->first->problem_state, 'fixed - council';
+    is $problem3->comments->first->text, '', 'correct comment text';
```

Comment:
This doesn't seem to be testing the translation any more, AIUI, so does the test need a more accurate name?